index.js

```diff
@@ -109,6 +113,24 @@ module.exports = (input, opts) => {
 		]);
 	}
 
+	if (runChecks) {
+		tasks.add([{
+			title: 'Ensure GitHub checks have passed',
+			skip: () => {
+				if (!pkg.repository) {
+					return 'No repository specified';
```

Comment:
> No repo specified in package.json

?

Reply:
What do you think about “No repository specified in package.json”? The `package.json` field is also called `repository` so I’m not sure whether we should abbreviate here.

Reply:
> “No repository specified in package.json”? The package.json field is also called repository so I’m not sure whether we should abbreviate here.

👍

Reply:
Done: c12e2726fc7512e932288f8c1c412b732030603d.

Reply:
Should np fallback to getting the repo URL from git itself? I feel like requiring `pkg.repository` is just needless busy work.

Reply:
I’ve had the same thought at one point. I don’t have particularly strong opinions in either direction.

On one hand, requiring `pkg.repository` encourages best practices (for example, [npmjs.com](https://www.npmjs.com) will display a link to the repository if the field is specified) and gives us higher assurance that it actually specifies the package’s repository (whereas the Git remote might be a fork). On the other hand, this task already kind of expects the Git remote to be the upstream repository (since we `git push` before querying the GitHub API).

Reply:
That's an interesting thought. If I'm on a fork and running `np`, I'm probably intentionally trying to publish some hacky thing I did, perhaps to a scoped package under my own personal namespace. I'm more likely to have a PR open on the `pkg.repository` (which I probably haven't updated and where CI checks are probably running) than to have CI setup on my fork. I don't think I have ever set up CI on a fork, though major projects obviously would. 🤔 

That being said, I still think if `pkg.repository` isn't defined at all, it might be worth having some kind of fallback. Just "thinking out loud" here.